project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -26,6 +26,7 @@ object Dependencies {
   private val TestContainersScalaVersion = "0.7.0"
   private val TypesafeConfigVersion = "1.3.1"
   private val VaultJavaDriverVersion = "3.0.0"
+  private val UUIDVersion = "3.1.3"
```

Comment:
Nitpick: Can you put this above `VaultJavaDriverVersion` to keep things in abc order?